<ui version="4.0" >
 <class>DeletedBookingClass</class>
 <widget class="QDialog" name="DeletedBookingClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NA10 DeletedBookingVC</string>
  </property>
  <widget class="QGroupBox" name="grpSearch" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>7</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Search Criteria:</string>
   </property>
   <widget class="QLabel" name="lbTo" >
    <property name="geometry" >
     <rect>
      <x>342</x>
      <y>23</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="articleNoSearch" >
    <property name="geometry" >
     <rect>
      <x>64</x>
      <y>21</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtFromSearch" >
    <property name="geometry" >
     <rect>
      <x>222</x>
      <y>21</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtToSearch" >
    <property name="geometry" >
     <rect>
      <x>362</x>
      <y>20</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch" >
    <property name="geometry" >
     <rect>
      <x>492</x>
      <y>19</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbArticleNo" >
    <property name="geometry" >
     <rect>
      <x>12</x>
      <y>23</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>ArticleNo.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFrom" >
    <property name="geometry" >
     <rect>
      <x>192</x>
      <y>23</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>from:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpDeletedBookings" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>73</y>
     <width>891</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title" >
    <string>Deleted Bookings:</string>
   </property>
   <widget class="QPushButton" name="btnRestore" >
    <property name="geometry" >
     <rect>
      <x>788</x>
      <y>411</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Restore</string>
    </property>
   </widget>
   <widget class="BasarTableWidget" name="tableWidget" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>21</y>
      <width>861</width>
      <height>381</height>
     </rect>
    </property>
    <property name="editTriggers" >
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount" >
     <number>11</number>
    </property>
    <property name="columnCount" >
     <number>14</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose" >
   <property name="geometry" >
    <rect>
     <x>810</x>
     <y>530</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
